tests/documents_tests.ts

```diff
@@ -499,3 +501,121 @@ describe.each([{ client: anonymousClient, permission: 'No' }])(
     })
   }
 )
+
+test(`Get request should not add double slash nor a trailing slash`, async () => {
+  try {
+    const res = await badHostClient.index(uidNoPrimaryKey.uid).getDocuments()
+    expect(res).toBe(undefined) // Left here to trigger failed test if error is not thrown
+  } catch (e) {
+    expect(e.message).toMatch(`${BAD_HOST}/indexes/movies_test/documents`)
+    expect(e.message).not.toMatch(`${BAD_HOST}/indexes/movies_test/documents/`)
+    expect(e.type).toBe('MeiliSearchCommunicationError')
+  }
+})
+
+test(`Get request should not add double slash nor a trailing slash`, async () => {
+  try {
+    const res = await badHostClient.index(uidNoPrimaryKey.uid).getDocuments()
+    expect(res).toBe(undefined) // Left here to trigger failed test if error is not thrown
+  } catch (e) {
+    expect(e.message).toMatch(`${BAD_HOST}/indexes/movies_test/documents`)
+    expect(e.message).not.toMatch(`${BAD_HOST}/indexes/movies_test/documents/`)
+    expect(e.type).toBe('MeiliSearchCommunicationError')
+  }
+})
```

Comment:
Not sure to see the difference between both of these tests

Reply:
Yep, my bad! I had doubled it to try out query parameters but it had none.